### ISSUE TITLE
10.12.2 will not npm install with node 7.6

### ISSUE THREAD

**Issue by mcrawshaw** [auth0/lock#901]
```
error auth0-lock@10.12.2: The engine "node" is incompatible with this module. Expected version "~6.9.1".
```
I can see an update to package.json that may have already fixed.  Would it be possible to release a new version?

Mark.

**Comment by fbascheper** [comment]
Shouldn't package.json be updated to
```
"engines": {
    "node": ">= 6.9.1",
 }
```

to support node v7, instead of just

`   "node": "^6.9.1"`

**Comment by hzalaz** [comment]
@fbascheper we only properly tested it with node 6 so We'd like to avoid unforeseen issues of using a new node major. We can update it once it's properly tested

**Comment by hzalaz** [comment]
@mcrawshaw it will be in next week release

**Comment by fbascheper** [comment]
OK, so you're supporting lock as a server-side application on node then? 

Using it like that has never come to my mind. My issue is that yarn no longer allows a dependency for my angular2 projects (client side only), because I have node7 installed on my dev machine. This means I'm now stuck to using old auth-lock versions, or revert to node6 locally. Yuck.

**Comment by luisrudge** [comment]
@fbascheper we're not supporting lock on node, but we need to make sure that you can clone this repo and you can build and test everything. This was only tested with node 6. I'll run some tests with node@7 and see if we can include that in the next release as well.

**Comment by fbascheper** [comment]
That would be awesome!

**Comment by evansiroky** [comment]
btw, we are using node 6, but Travis seems to use 6.10.0 by default which is not compatible with `~6.9.1`.

**Comment by noah79** [comment]
This is starting to bite us.  We only use auth0-lock in our client, but now our universal app is unable to build without downgrading node.  Not good.

**Comment by luisrudge** [comment]
@noah79 did you upgrade your lock version? We fixed this in #909
